x-pack/plugins/apm/server/lib/transactions/charts/get_avg_response_time_anomalies/__test__/get_anomaly_aggs.test.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// @ts-ignore
+import { getAnomalyAggs } from '../get_anomaly_aggs';
+
+test('getAnomalyAggs should swallow errors', () => {
+  const failClient = jest.fn(() =>
+    Promise.reject(new Error('anomaly lookup failed'))
+  );
+
+  expect(getAnomalyAggs({ client: failClient })).resolves.toEqual(null);
```

Comment:
Async jest tests are crazy deceiving. You need to `return` or `await` your expectation when doing this.

```suggestion
  return expect(getAnomalyAggs({ client: failClient })).resolves.toEqual(null);
```

The following will for instance still pass:
```js
    const failClient = jest.fn(() => Promise.resolve('I am for sure not null'));
    expect(getAnomalyAggs({ client: failClient })).resolves.toEqual(null);
```

Reply:
Ahhh yes, I used to be so good about this but switched to async/await for all my tests and then forgot about it. Will fix, good catch.